hack/dev-setup

```diff
@@ -214,4 +219,56 @@ OaMnRwirobPupceHD5zvuGPjjDaPhc1yeRMYo+xXgLL+mYyXR1ZYGuI=
 -----END RSA PRIVATE KEY-----
 EOF
 
+cat <<EOF > "${DEV_DIR}/tls/gardener-apiserver.crt"
```

Comment:
Why is this needed?

Reply:
I added these files to be able to use them inside my debug configurations (I cannot fetch the certificates like in the `start` scripts there).
But you are right, this is not needed. I can simply use paths to the files already present inside `hack/local-garden`, like `hack/local-garden/certificates/certs/default-admin.crt` for my debug configs.
So it removed it from `hack/dev-setup`, PTAL :)